client/pages/registration.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+
+class Registration extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: '',
+      userName: '',
+      firstName: '',
+      lastName: '',
+      email: '',
+      password: ''
+
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange1 = this.handleChange1.bind(this);
+    this.handleChange2 = this.handleChange2.bind(this);
+    this.handleChange3 = this.handleChange3.bind(this);
+    this.handleChange4 = this.handleChange4.bind(this);
+    this.handleChange5 = this.handleChange5.bind(this);
+  }
+
+  handleSubmit(event) {
+    /* eslint-disable no-console */
+    console.log(this.state);
+    event.preventDefault();
+  }
+
+  handleChange(event) {
+    this.setState({
+      value: event.target.value
+    });
+  }
+
+  handleChange1(event) {
+    this.setState({
+      userName: event.target.value
+    });
+  }
+
+  handleChange2(event) {
+    this.setState({
+      firstName: event.target.value
+    });
+  }
+
+  handleChange3(event) {
+    this.setState({
+      lastName: event.target.value
+    });
+  }
+
+  handleChange4(event) {
+    this.setState({
+      email: event.target.value
+    });
+  }
+
+  handleChange5(event) {
+    this.setState({
+      password: event.target.value
+    });
+  }
```

Comment:
This should be combined into a single `handleChange` method. If you add a `name` attribute to each input set to the value that matches the property name in state you can do the following:

```JSX
handleChange(event) {
  const name = event.target.name;
  const value = event.target.value;

  this.setState({
    [name]: value
  });
}
```